src/store/ui/actions.ts

```diff
@@ -104,6 +106,15 @@ export function disconnectAnchor(
   };
 }
 
+export function resetAll(): AppThunk<void> {
+  return (dispatch) => {
+    dispatch({
+      type: UI_RESET_ALL,
```

Comment:
We probably want a `SIDENOTES` in there. These actions could conflict with other apps.